prow/e2e-suite.sh

```diff
@@ -91,4 +94,4 @@ E2E_ARGS+=("--skip_cleanup")
 time ISTIO_DOCKER_HUB=$HUB \
   E2E_ARGS="${E2E_ARGS[*]}" \
   JUNIT_E2E_XML="${ARTIFACTS_DIR}/junit.xml" \
-  make with_junit_report TARGET="${SINGLE_TEST}" ${E2E_TIMEOUT:+ E2E_TIMEOUT="${E2E_TIMEOUT}"}
+  make with_junit_report TARGET="${SINGLE_TEST}" ${VARIANT:+ VARIANT="${VARIANT}"} ${E2E_TIMEOUT:+ E2E_TIMEOUT="${E2E_TIMEOUT}"}
```

Comment:
`:+` is a new one for me, I learn something new everyday in our bash scripts :slightly_smiling_face:

Reply:
We weren't aware of this either, but we copied it from the `E2E_TIMEOUT` argument to keep things consistent.